src/ageml/visualizer.py

```diff
@@ -133,17 +136,15 @@ def features_vs_age(self, X: list, Y: list, corr: list, order: list, markers,
                 title += "\n$\\rho_{%s}$: %s%.3f" % (label, markers[n][o], corr[n][o])
             ax.set_title(title)
             ax.legend(labels)
-            plt.suptitle(f"Features vs. Age\n{name}", y=0.99)
+            plt.suptitle(f"Features vs. Age\n{tag.system}", y=0.99)
         plt.tight_layout()
 
-        if name == "":
-            filename = "features_vs_age.png"
-        else:
-            filename = f"features_vs_age_{name}.png"
+        # Save file
+        filename = f"features_vs_age_controls{'_'+tag.system if tag.system != '' else ''}.png"
```

Comment:
Missing whitespace around '+' operator. Linter will catch it in the future. Good use of the NameTag object. I like it